### ISSUE TITLE
Program keeps crashing

### ISSUE THREAD

**Issue by DBronx** [Hyrules/WinHue#261]
[WinHue3.log](https://github.com/Hyrules/WinHue/files/3745803/WinHue3.log)
Before submitting a request please make sure you are running the latest version of WinHue 
available in the release section of this site. Support will not be done on older version.

### Your setup 

WinHue version : (eg. 3.0.2400.0)
Bridge api version : (1.16)
Light model : (if you have a problem with a particular light)

### Expected behavior and actual behavior
Cannot perform any action
...

### Actual behavior
Program crashes when **any** action attempted
...

### Steps to reproduce the problem

1) ...Click on "search for new lights"
2) ...Hover over Edit menu
3) ...

In the case of a crash, please make sure to add a logfile containg the crash located in the %APPDATA%\WinHue3
to the issue you are opening.

**Comment by Hyrules** [comment]
hmm... that's really odd. Why are you getting and unauthorized user error ? Looks like your user has somehow been removed from the whitelist (cannot be done inside winhue since philips require to do it in their web gui) so someone mostlikely deleted your api key. This is either on purpose of trying to crash the app or someone really deleted your key. You can test your api key by going in a browser and typeing "http://<your bridge ip>/api/<your api key>/lights" without the quotes and replacing the required sections. This will give you either an unauthorized error or list the bridge lights.

**Comment by DBronx** [comment]
I'm sorry, I don't understand...

When I try to go to http:///api//lights   - I get a page not found error.

Jeff

**Comment by Hyrules** [comment]
http://[your bridge ip]/api/[your api key]/lights github just removed everything between "<>" you have you have to add the bridge ip after http:// and before /api and add the apikey after /api/ before lights.

**Comment by Hyrules** [comment]
try this one : http://192.168.1.24/api/V496AoY55e7PAvZoSzJ9usthLp4I6Wu4rlZdi1HG/lights

**Comment by DBronx** [comment]
I've tried the IP address of the bridge and //lights (e.g.
192.168.x.x//lights) and get a 404 error

**Comment by Hyrules** [comment]
no this wont work you really need the full url with the api key. Check my last message you should have it.

**Comment by DBronx** [comment]
This is what I got...

[{"error":{"type":1,"address":"/lights","description":"unauthorized user"}}]

**Comment by Hyrules** [comment]
Yeah basically you api key have been revoked. Open window explorer and paste this %APPDATA%\WinHue in the address bar up top. You will see some WinHue files. Delete the WinHueBridges.set and do a new pairing.

**Comment by DBronx** [comment]
That did the trick !!!

Thank you so much and thanks for the work on the app, can't wait to get
cracking.

Jeff

**Comment by Hyrules** [comment]
Great to see you go that working. Don't hesitate I will try to fix this issue on my side because this could happen but should not and it should definitely not crash the app.
